artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master.TaskGeneration/Features/GenerateTasks/GenerateTasksRequest.cs

```diff
@@ -0,0 +1,7 @@
+namespace Artificial.Scrum.Master.TaskGeneration.Features.GenerateTasks;
+public readonly record struct GenerateTasksRequest
```

Comment:
skoro to rekord to nie definiuj tak propert. defaultowo rekord jest immutable więc property są init only.

Reply:
public readonly record struct GenerateTasksRequest(
    int Prop1,
    string Prop2,...);